libs/permissions/permission-management-cache-impl/src/main/kotlin/net/corda/libs/permissions/management/cache/impl/PermissionManagementCacheImpl.kt

```diff
@@ -66,6 +66,19 @@ internal class PermissionManagementCacheImpl(
         return permissions[permissionId]
     }
 
+    override fun getUsersByProperty(propertyKey: String, propertyValue: String): List<User>? {
```

Comment:
I think we can change the signature of this method to return an empty `Set` when none are found. @aadhavskthvl - WDYT?
```suggestion
    override fun getUsersByProperty(propertyKey: String, propertyValue: String): Set<User> {
```